Simplenote/simplenote_app_credentials_demo.json

```diff
@@ -0,0 +1,14 @@
+{
+  "appbot_key": "not-required",
+  "bithockey_id": "not-required",
+  "google_analytics_id": "not-required",
+  "itunes_app_id": "not-required",
+  "itunes_review_url": "not-required",
+  "sentry_dsn": "not-required",  
```

Comment:
We'll need something that looks like a valid DSN here. 

If we wind up merging the templates, we could get rid of this file entirely – the `not-required` bits could just become comments in the templates.